Tests/SnapshotTestingTests/__Snapshots__/InlineSnapshotTests/testUpdateSnapshotWithShorterExtendedDelimiter2.1.swift

```diff
@@ -1,13 +1,17 @@
 import XCTest
+
 @testable import SnapshotTesting
+
 extension InlineSnapshotsValidityTests {
   func testUpdateSnapshotWithShorterExtendedDelimiter2() {
     let diffable = #######"""
-    \"""#
-    """#######
+      \"""#
+      """#######
 
-    _assertInlineSnapshot(matching: diffable, as: .lines, with: ##"""
-    \"""#
-    """##)
+    _assertInlineSnapshot(
```

Comment:
dang, we may need to either make `_assertInlineSnapshots` more resilient or ignore test files from formatting :/